packages/snap-preact-components/src/components/Atoms/LoadMore/readme.md

```diff
@@ -0,0 +1,123 @@
+# LoadMore
+
+Renders a "Load More" button that displays the next page of results. Recommended to be used with SearchController's `infinite` config
+
+## Sub-components
+- Button
+
+## Usage
+
+### pagination / controller
+The required `pagination` (or `controller`) prop specifies a reference to the pagination store object.
+
+```jsx
+<LoadMore pagination={controller.store.pagination} />
+```
+
+```jsx
+<LoadMore controller={controller} />
+```
+
+### auto
+The `auto` prop will not render a "Show More" button and instead fetch the next page of results when the component appears in the viewport. Recommended to place underneath Results to allow for infinite scrolling. 
```

Comment:
thoughts on renaming this to be a little more specific? "auto" could mean so mean anything and without looking at the docs its hard to know what it does. 

maybe autoFetch ? or autoLoadMore?

Reply:
Renamed to autoFetch